test/components/tabTest.js

```diff
@@ -15,6 +16,53 @@ describe('tabs', function () {
       .waitForVisible(urlInput)
   }
 
+  
+  describe('back forward actions', function () {
+    Brave.beforeAll(this)
+    before(function * () {
+      yield setup(this.app.client)
+    })
+
+    it('sets correct title', function * () {
+      var page1 = Brave.server.url('page1.html')
+      var page2 = Brave.server.url('page2.html')
+
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(page1)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForExist('.tab.active')
+        .getText('.tab.active .tabTitle')
+        .then(function(text) {
+          assert.equal('Page 1', text)
+        })
+        .tabByIndex(0)
+        .loadUrl(page2)
+        .windowByUrl(Brave.browserWindowUrl)
+        .waitForExist('.tab.active')
```

Comment:
I think you have a race condition with these checks because `.tab.active` already exists from the previous load. You could put the getText and assert equal in a `waitUntil` block and that should be more reliable